src/components/forms/schedule-settings-form/pre-filter-input.js

```
@@ -4,11 +4,9 @@ import LevelDropdown from "../../inputs/level-dropdown";
 import EventTypeDropdown from "../../inputs/event-type-dropdown";
 import TrackDropdown from "../../inputs/track-dropdown";
 import TrackGroupDropdown from "../../inputs/track-group-dropdown";
-import {
-  CompanyInput,
-  SpeakerInput,
-  TagInput
-} from "openstack-uicore-foundation/lib/components";
+import CompanyInput from "openstack-uicore-foundation/lib/components/inputs/company-input"
```

ESLint

[error] `openstack-uicore-foundation/lib/components/inputs/company-input` import should occur before import of `../../../actions/schedule-settings-actions`

(import/order)